### DISCUSSION TITLE
Does DMC5 VR mode work with co-op mod?

### DISCUSSION THREAD

**Discussion by harbingerthrace** [praydog/REFramework#575] [General Questions]
Does @praydog 's DMC5 VR mode work with co-op mod?  I think this is the main co-op mod? 

https://www.nexusmods.com/devilmaycry5/mods/353

Thanks in advance for your response! =)

**Comment by harbingerthrace** [comment]
Kinda bummed I can't get an answer to this =(.  @praydog any idea?

**Comment by praydog** [comment]
I don't know why don't you try it? I've never used it.
